spec/controllers/file_controller_spec.rb

```diff
@@ -58,6 +80,19 @@
       get :show, params: { id: 'xf680rd3068', file_name: 'xf680rd3068_1.jp2', download: 'any' }
     end
 
+    context 'when Stanford restricted' do
+      before do
+        stub_rights_xml(stanford_restricted_rights_xml)
+        allow(Purl).to receive(:public_json).and_return(stanford_json)
+      end
+
+      it 'sends CORs header' do
+        subject
+        expect(response.headers.to_h).to include 'Access-Control-Allow-Origin' => 'https://embed.stanford.edu',
+                                                 'Access-Control-Allow-Credentials' => 'true'
+      end
+    end
+
```

Comment:
Is it worth adding a test for when the file is not Stanford Only to make sure the header is `*`?

Reply:
Sure, I can take a look.

Reply:
I see that CI is running the tests like this, which fails for me locally (unlike a regular rspec run):

    SETTINGS__FEATURES__COCINA=true bin/rake spec

Reply:
Ahh, yes.  good catch.  likely an issue with the mocking in the test not accounting for cocina rights computation

Reply:
Fixed. Had a problem in my mocked cocina json (it wasn't internally consistent for stanford only).